Wiki for out-of-ammo sounds (10693) (#98)
endless-sky/endless-sky-wiki: wiki/CreatingOutfits.md

* `sound`: a path to a sound, relative to the "sounds" folder, and not including the extension or the specifiers (e.g. "laser", not "sounds/laser~.wav"). The sound file must be a mono (not stereo) WAV file with 16-bit, 44100 Hz encoding. Adding `~` to the file name makes it loop. Since **v. 0.10.11**, you can also provide an `@3x` sound that is played in fast-forward mode (e.g. `laser@3x~.wav`).

>  * `"empty sound"`: a path to a sound (see the requirements above) that is played when the player attempts to fire a weapon with unmet ammo or fuel requirements. **(v. 0.10.13)**

Reviewer: ```suggestion
 * `"empty sound"`: a path to a sound (see the requirements above) that is played when the player attempts to fire a weapon that does not have enough ammo or fuel to fire. **(v. 0.10.13)**
```
